articles/quickstart/webapp/aspnet-core/_includes/_login.md

```diff
@@ -142,44 +121,9 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 }
 ```
 
-### Obtain an Access Token for Calling an API
-
-If you want to call an API from your MVC application, you need to obtain an Access Token issued for the API you want to call. To obtain the token, pass an additional `audience` parameter containing the API identifier to the Auth0 authorization endpoint.
-
-In the configuration for the `OpenIdConnectOptions` object, handle the `OnRedirectToIdentityProvider` event and add the `audience` parameter to `ProtocolMessage`.
-
-```csharp
-// Startup.cs
-
-public void ConfigureServices(IServiceCollection services)
-{
-    // Some code omitted for brevity...
+## Trigger Authentication
```

Comment:
Moved the order here to be in line with https://auth0.com/docs/quickstart/webapp/aspnet-core-3, which talks about Triggering Authentication before Calling an API.